core/src/main/java/fr/sncf/osrd/cli/ApiServerCommand.java

```diff
@@ -55,7 +55,7 @@ public int run() {
         var authorizationToken = System.getenv("FETCH_INFRA_AUTHORIZATION");
         var middleWareBaseUrl = getMiddlewareBaseUrl();
         var httpClient = new OkHttpClient.Builder().readTimeout(120, TimeUnit.SECONDS).build();
-        var infraManager = new InfraManager(middleWareBaseUrl, authorizationToken, httpClient);
+        var infraManager = new InfraManager(middleWareBaseUrl, authorizationToken, httpClient, false);
```

Comment:
my understanding is that this change causes the back-end to fail if the infrastructure isn't loaded; is that correct? if so, is editoast ready to handle it? does it start the download beforehand?

Reply:
I've checked, and thanks to Leo's work on migrating the endpoints, editoast returns the correct 400 error when Core returns a 400 error! 😄